### ISSUE TITLE
Server & Client not running ,already installed dateutil

### ISSUE THREAD

**Issue by BSGILL12** [insidesherpa/JPMC-tech-task-1-PY3#77]
![Screenshot (262)](https://user-images.githubusercontent.com/47722450/81745967-542d2600-94c3-11ea-81ef-21e6eed36fc7.png)

**Comment by JoY-cmd** [comment]
According to the error, I can see that you are directing a wrong path. Check which Python version you are using and execute the same command lines as mentioned in the Document. Hope it helps.

**Comment by BSGILL12** [comment]
> According to the error, I can see that you are directing a wrong path. Check which Python version you are using and execute the same command lines as mentioned in the Document. Hope it helps.

Yes it resolve my server error Please have look on client,python client3.py have not been resolved,When http 8080 started n server then create new window for client but it not run

**Comment by BSGILL12** [comment]
![Screenshot (263)](https://user-images.githubusercontent.com/47722450/81864495-59a17380-958a-11ea-907a-b4466fb91c07.png)
Have Look For error on Client 's cmd window

**Comment by Adithya-001** [comment]
![Screenshot (113)](https://user-images.githubusercontent.com/62662281/81977430-a303c880-9647-11ea-85c5-11ce1daab4f1.png)
I am getting the same fault, wherein although the command is executed nothing is happening could you please guide me

**Comment by dheerajodha** [comment]
Hi @BSGILL12  and @Adithya-001 ,
Please make sure that you run python server.py and python client.py commands in **two seperate CMD**

Run python server.py FIRST followed by python client.py

If you get some more errors, then post your error screenshots here.
If you've already solved your issue, please close this issue, thanks!

**Comment by Adithya-001** [comment]
Thank you that helped.

**Comment by Evelinmaria** [comment]
How much time it will take to establish server.py and client.py in the system?

**Comment by dheerajodha** [comment]
@Evelinmaria  I'm not sure what you mean by "establish", but it won't take much time to start server and client.
